### PR TITLE
Skeleton

### PR DESCRIPTION
**Pull request by katie-quesnel13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ _Have another more specific idea? You may want to check out our vibrant collecti
 
 1.  **Start developing.**
 
-    Navigate into your new site’s directory and start it up.
```
Comment (potatowave) [inline README.md:28]: Is this some find/replace junk?

Comment (katie-quesnel13) [reply README.md:28]: I've not done anything on the readme: i think it's just part of the default page? i'm not sure

Comment (katie-quesnel13) [reply README.md:28]: looking at it, probably an accidental replace while i was looking for the keyword

File: package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "cypress": "^4.6.0",
     "cypress-axe": "^0.8.1",
     "gatsby": "^2.22.0",
+    "gatsby-cli": "^2.12.43",
```
Comment (potatowave) [inline package.json:15]: This shouldn't be present at all.  Gatsby CLI is only required on the develop machine.

File: README.md (hunk 1 above)
Comment (potatowave) [reply README.md:28]: This kind of mistake can be avoided by only intentionally adding files when committing.

E.g. don't use `git commit .`.  It's better to run `git status` to review the changes you want to commit (use `git diff <file>), and then add each file you want to commit.  That way if there's an unintentional edit, like the one here, it never gets committed.

File: package.json (hunk 2 above)
Comment (potatowave) [reply package.json:15]: If you revert these changes:

- Edits to README.md
- Edits to package.json
- Edits to yarn.lock

I can approve this pull request.

**Pull request merged**